protocol/x/subaccounts/keeper/subaccount_test.go

```diff
@@ -2499,23 +2820,16 @@ func TestUpdateSubaccounts(t *testing.T) {
 			}
 
 			for i, p := range tc.perpetuals {
-				perp, err := perpetualsKeeper.CreatePerpetual(
+				perpetualsKeeper.SetPerpetual(
```

Comment:
Use `SetPerpetual` so input perpetual with OI is directly initialized (`CreatePerpetual` ignores OI)